accelerator/main.c

```diff
@@ -40,6 +40,21 @@ void health_track(void* arg) {
       }
     }
 
+    char uuid[UUID_STR_LEN] = {};
+    // The usage exceeds the maximum redis capacity
+    while (core->cache.cache_capacity < cache_occupied_space()) {
+      cache_server_lock();
```

Comment:
Why the lock only appears here?
What about other ones that use the same cache server?

Reply:
I did have the same question as @YingHan-Chen .